### DISCUSSION TITLE
Please make more demo videos

### DISCUSSION THREAD

**Discussion by gaborho** [rxhanson/RectanglePro-Community#305] [General]
Hi,

I learned that you started to publish demo/tutorial videos @ https://www.youtube.com/@rectangleapp
I found it very useful. I kindly ask you to create more of such videos. I feel that the app has really a lot of features, but they are hard to grasp based purely on website "documentation", and the name of the configurable options in the GUI sometimes not represent what it does. I feel that only by experimenting with the options I can understand what the feature is about. Thanks for your efforts.

**Comment by rxhanson** [comment]
Thanks for the feedback. Indeed, I have plans to make videos for each tab in the preferences window. My schedule has been maxed out for the past few months, and I'm hoping to get some more time for this in the near future.

**Reply by quinncomendant** [reply]
I too want more tutorial videos! I just started using Rectangle Pro and am overwhelmed with all the options. 🫠

**Comment by pinkgalaxy** [comment]
> Hi,
> 
> I learned that you started to publish demo/tutorial videos @ https://www.youtube.com/@rectangleapp I found it very useful. I kindly ask you to create more of such videos. I feel that the app has really a lot of features, but they are hard to grasp based purely on website "documentation", and the name of the configurable options in the GUI sometimes not represent what it does. I feel that only by experimenting with the options I can understand what the feature is about. Thanks for your efforts.

What about scheduling a YouTube live to demonstrate the different features a Rectangle Pro and how to use them? That way, you show it once and use that video for reference in the future.

**Comment by pinkgalaxy** [comment]
Here is my attempt at a demo video for those who'd find it helpful:

SNAP PANEL:
https://github.com/rxhanson/RectanglePro-Community/assets/164553001/d10a648e-fc89-40f4-ba89-340584ac5a9f

**Comment by pinkgalaxy** [comment]
WINDOW THROW:
https://github.com/rxhanson/RectanglePro-Community/assets/164553001/91947692-6db6-40c9-bafc-e6889b1c3827

**Comment by pinkgalaxy** [comment]
STASH:

https://github.com/rxhanson/RectanglePro-Community/assets/164553001/9f27415a-6522-4213-95ba-f363f29891bd
